src/ostree/ot-builtin-fsck.c

```diff
@@ -313,7 +312,9 @@ ostree_builtin_fsck (int argc, char **argv, OstreeCommandInvocation *invocation,
 
       ostree_object_name_deserialize (serialized_key, &checksum, &objtype);
 
-      if (objtype == OSTREE_OBJECT_TYPE_COMMIT)
+      g_assert (objtype == OSTREE_OBJECT_TYPE_COMMIT);
+
+      if (TRUE)  // Temporarily avoiding the noise of de-indenting this whole thing
```

Comment:
I'd personally vote for taking the re-indentation noise since I don't think this is going to be temporary. Whether it's `list_commits_starting_with` now or `list_objects_set` later, I think this program is going to stick with some method that only scans for commit objects (with or without the above `assert`). Not a blocker, though.